File: evidence/generated-systems/golang/governance-on-golang/governance-on-golang/internal/model/Person.go
package model

import (
    "gorm.io/gorm"
)

//==============================================================
// Person Declaration
//==============================================================
type Person struct {
    gorm.Model
     FirstName                                    string
    LastName                                    string
    Email                                                            string
    Department                                    string
     RoleAssignments           []RoleAssignment `gorm:"foreignKey:RoleAssignmentsFromPersonId"`
     OwnedPolicies           []Policy `gorm:"foreignKey:OwnedPoliciesFromPersonId"`
     CorrectiveActions           []CorrectiveAction `gorm:"foreignKey:CorrectiveActionsFromPersonId"`

// parent associations as their child

}

File: evidence/generated-systems/rubyonrails/governance-on-rails/governanceonrails/app/models/person.rb
class Person < ApplicationRecord


  composed_of :emailAddress,
    class_name: "EmailAddress",
    mapping: [
      %w[emailAddress_value value]
    ]

  has_many :RoleAssignments, class_name: 'RoleAssignment'
  has_many :OwnedPolicies, class_name: 'Policy'
  has_many :CorrectiveActions, class_name: 'CorrectiveAction'

end
